src/eureka-client.js

```diff
@@ -204,18 +205,16 @@ export class Eureka {
     if (!registry) {
       throw new Error('Unable to transform empty registry');
     }
-    if (registry.length) {
```

Comment:
Sweet. I'll push a patch version to npm after that so we have a working module on npm again :P